### PR TITLE
Print a final report suggesting to open an issue on rust repo

### PR DESCRIPTION
**Pull request by spastorino**
This PR make the tool output a final report.

The report looks like ...

```
==================================================================================
= Please open an issue on Rust's github repository                               =
= https://github.com/rust-lang/rust/issues/new                                   =
= Below you will find a text that would serve as a starting point of your report =
==================================================================================

# Regression found in the compiler

searched nightlies: from nightly-2019-09-29 to nightly-2019-09-30
regressed nightly: nightly-2019-09-30
searched commits: from https://github.com/rust-lang/rust/commit/488381ce9ef0ceabe83b73127c659e5d38137df0 to https://github.com/rust-lang/rust/commit/8431f261dd160021b6af85916f161a13dd101ca0
regressed commit: https://github.com/rust-lang/rust/commit/0bbab7d99dde8620604fb265706dc8bff20345a7
source code: URL OF A REPOSITORY THAT REPRODUCES THE ERROR

## Instructions

PLEASE GIVE US INSTRUCTIONS ON WHAT YOUR APPLICATION DOES
## Error

<details><summary>COLLAPSIBLE ERROR STACKTRACE</summary>
<p>

PASTE THE ERROR YOUR APP GIVES HERE (run with RUST_BACKTRACE=1)

</p></details>
```

And that pasted on an issue looks like https://github.com/spastorino/cargo-bisect-sample/issues/1

r? @Mark-Simulacrum

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -869,6 +870,78 @@ fn print_results(cfg: &Config, client: &Client, bisection_result: &BisectionResu
...
+        ci_toolchains[*ci_found],
+    );
+
+    eprintln!("source code: URL OF A REPOSITORY THAT REPRODUCES THE ERROR");
+
+    eprintln!("");
+
+    eprintln!("## Instructions");
+    eprintln!("");
+    eprintln!("PLEASE GIVE US INSTRUCTIONS ON WHAT YOUR APPLICATION DOES");
```
Comment (Mark-Simulacrum) [inline src/main.rs]:
```suggestion
    eprintln!("Please give the steps for how to build your repository (platform, system dependencies, etc.)");
```

File: src/main.rs [hunk 2]
```diff
@@ -869,6 +870,78 @@ fn print_results(cfg: &Config, client: &Client, bisection_result: &BisectionResu
...
+
+    eprintln!("## Error");
+    eprintln!("");
+    eprintln!("<details><summary>COLLAPSIBLE ERROR STACKTRACE</summary>");
+    eprintln!("<p>");
+    eprintln!("");
+    eprintln!("```bash");
+    eprintln!("PASTE THE ERROR YOUR APP GIVES HERE (run with RUST_BACKTRACE=1)");
+    eprintln!("```");
```
Comment (Mark-Simulacrum) [inline src/main.rs]: This section is a bit confusing -- what do we mean by "error your app gives"? I would maybe just remove it?

Comment (spastorino) [reply src/main.rs]: I meant to paste the output of the error the (regressed) compiler is giving.

File: src/main.rs [hunk 3]
```diff
@@ -869,6 +870,78 @@ fn print_results(cfg: &Config, client: &Client, bisection_result: &BisectionResu
...
+    eprintln!("");
+    eprintln!("Please give the steps for how to build your repository (platform, system dependencies, etc.)");
+
+    eprintln!("## Error");
+    eprintln!("");
+    eprintln!("<details><summary>COLLAPSIBLE ERROR STACKTRACE</summary>");
+    eprintln!("<p>");
+    eprintln!("");
+    eprintln!("```bash");
+    eprintln!("PASTE THE ERROR YOUR APP GIVES HERE (run with RUST_BACKTRACE=1)");
```
Comment (spastorino) [inline src/main.rs]:
```suggestion
    eprintln!("Paste the error the compiler is giving");
```

**Pull request merged**